pkg/operator/ceph/csi/spec.go

```diff
@@ -305,6 +308,11 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 		tp.EnableVolumeReplicationSideCar = true
 	}
 
+	tp.EnableCSIAddonsSideCar = false
+	if strings.EqualFold(k8sutil.GetValue(r.opConfig.Parameters, "CSI_ENABLE_CSI_ADDONS", "false"), "true") {
```

Comment:
We would also need to add this setting to several other places:
- Manifests: operator.yaml, operator-openshift.yaml
- Helm chart: deployment.yaml, values.yaml
- Documentation: helm-operator.md

Reply:
Ack!

Reply:
Additionally, `ceph-upgrade.md` lists the default CSI images in 2 places. Please also update this doc.

Reply:
done